python/taichi/lang/ast/ast_transformer.py

```diff
@@ -1178,10 +1178,6 @@ def build_Continue(ctx, node):
     def build_Pass(ctx, node):
         return None
 
-    @staticmethod
-    def build_Raise(ctx, node):
-        raise build_stmt(ctx, node.exc)
```

Comment:
Can we add a test to show what's the current err msg when users write Raise in their kernel? Thanks!